test/gui/shared/steps/steps.py

```diff
@@ -1333,3 +1380,4 @@ def step(context, username):
         filename = syncPath + row[0]
         f = open(join(syncPath, filename), "w")
         f.close()
+    test.xvp("VP_VFS_enabled")
```

Comment:
Why do you need xvp here? FYI: The `xvp` is used to test the visual verification that is expected to fail.